js/views/PageNav.js

```diff
@@ -4,7 +4,7 @@ import { View } from 'backbone';
 import loadTemplate from '../utils/loadTemplate';
 import app from '../app';
 import $ from 'jquery';
-import SettingsModal from './modals/settings/Settings';
+import SettingsModal from './modals/Settings/Settings';
```

Comment:
Do I need my server running to see this?

On Aug 11, 2016 10:03 AM, "Robert Misiorowski" <notifications@github.com>
wrote:

> In js/views/PageNav.js
> <https://github.com/OpenBazaar/openbazaar-desktop/pull/28#discussion_r74461487>
> :
>
> > @@ -4,7 +4,7 @@ import { View } from 'backbone';
> >  import loadTemplate from '../utils/loadTemplate';
> >  import app from '../app';
> >  import $ from 'jquery';
> > -import SettingsModal from './modals/settings/Settings';
> > +import SettingsModal from './modals/Settings/Settings';
>
> I wouldn't think this folder should be capitalized. Maybe we can change it
> in the file structure? Do you have strong feelings that it should be
> capitalized?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenBazaar/openbazaar-desktop/pull/28/files/2c62789f2b3742a2e5b28235bf5dd1cf1281380b..272669532ff014ca1fb595a4786272f0fc7f30f5#r74461487>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMdihhkdkXjUSn9071kcUcPzIigbGH26ks5qe1XcgaJpZM4JiTFy>
> .
>